irohad/main/irohad.cpp

```diff
@@ -42,8 +42,8 @@ bool validate_config(const char *flag_name, std::string const &path) {
  * Gflag valigator.
  * Validator for the keypair files path input argument.
  * Path is considered to be valid if it is not empty.
- * @param flag_name - flag name
- * @param path      - file name
+ * @param flag_name - flag name. Must be 'keypair' in this case
+ * @param path      - file name. Should be path to the keypair files
```

Comment:
Why params has different modals btw (must/should)?

Reply:
Because flag name can only have one value since it is hardcoded. But the value can has anything inside.